tests/test_indices.py

```diff
@@ -469,6 +541,46 @@ def test_1d(self, tasmax_series, tasmin_series):
         np.testing.assert_allclose(hwml.values, 0)
 
 
+class TestHotSpellFrequency:
+    def test_1d(self, tasmax_series):
```

Comment:
Maybe that could be a `pytest.mark.parametrize` case?

Reply:
Done.